coalib/testing/LocalBearTestHelper.py

```diff
@@ -186,9 +204,10 @@ def check_results(self,
                                   tempfile_kwargs=tempfile_kwargs,
                                   settings=settings)
         if not check_order:
-            self.assertEqual(sorted(bear_output), sorted(results), msg=msg)
+            self.assertComparableObjectsEqual(
+                sorted(bear_output), sorted(results))
         else:
-            self.assertEqual(bear_output, results, msg=msg)
```

Comment:
Because it was a bit difficult to implement the messages in a modular way, we shifted that for consideration into Phase 2 of GSoC. We will also discuss the behaviour and improval of the general assertion there, for cases where `len(A) != len(B)` :+1: